test/unit/components/next-step-button.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {mountWithIntl} from '../../helpers/intl-helpers.jsx';
+import NextStepButton from '../../../src/components/join-flow/next-step-button';
```

Comment:
Travis is complaining that React and NextStepButton aren't used. But obviously NextStepButton is used! Not sure what's necessary to get the linter to see this. I was seeing linting errors make a similar mistake before, but it seemed to clear up when the lint versions were updated.

Reply:
We didn't quite have the right react eslint config and my file was a .js file instead of a .jsx file.... Fixed now though.